components/resources/brave_components_resources.grd

```diff
@@ -45,11 +45,16 @@
       </if>
 
       <!-- WebUI wallet resources -->
-      <include name="IDR_WALLET_PAGE_HTML" file="../brave_wallet_ui/page/wallet_page.html" type="BINDATA" />
+      <if expr="is_ios">
+        <include name="IDR_WALLET_PAGE_HTML" file="../brave_wallet_ui/page/wallet_page_ios.html" type="BINDATA" />
+      </if>
+      <if expr="not is_ios">
+        <include name="IDR_WALLET_PAGE_HTML" file="../brave_wallet_ui/page/wallet_page.html" type="BINDATA" />
+      </if>
       <include name="IDR_BRAVE_WALLET_NFT_DISPLAY_HTML" file="../brave_wallet_ui/nft/nft.html" type="BINDATA" />
       <include name="IDR_BRAVE_WALLET_MARKET_DISPLAY_HTML" file="../brave_wallet_ui/market/market.html" type="BINDATA" />
       <include name="IDR_BRAVE_WALLET_LINE_CHART_DISPLAY_HTML" file="../brave_wallet_ui/line_chart/line_chart.html" type="BINDATA" />
-      <if expr="not is_android">
+      <if expr="not is_android and not is_ios">
         <include name="IDR_WALLET_PANEL_HTML" file="../brave_wallet_ui/panel/wallet_panel.html" type="BINDATA" />
         <include name="IDR_BRAVE_WALLET_LEDGER_BRIDGE_HTML" file="../brave_wallet_ui/ledger/ledger.html" type="BINDATA" />
         <include name="IDR_BRAVE_WALLET_TREZOR_BRIDGE_HTML" file="../brave_wallet_ui/trezor/trezor.html" type="BINDATA" />
```

Comment:
Could you please refactor wallet section in this format:
```
<if expr="is_win or is_macosx or is_linux">
...
</if>
<if expr="is_android">
...
</if>
<if expr="is_ios">
...
</if>
```
It would be much clearer what resources each platform gets. It's ok to have some `<include ..>` lines duplicated.